README.rst

```diff
@@ -51,6 +52,59 @@ Accepts an open-ended set of paired lists, each expected to provide an estimate
   19866960, 5437.757350231803
 
 
+Approximating means
+~~~~~~~~~~~~~~~~~~~
+
+Estimate a mean and approximate the margin of error. Uses a simulation approach. 
+First the number of units in each bin is simulated (assuming a normal distribution around the estimate). 
+Then for each unit a value within the bin is simulated (assuming a uniform distribution within each bin).
+We can then calculate the mean directly from the simulated data. We repeat the simulation many times to calculate an empirical margin of error.
+
+Expects a list of dictionaries that divide the full range of data values into continuous categories. Each dictionary should have four keys:
+
+.. list-table::
+  :header-rows: 1
+
+  * - key
+    - value
+  * - min
+    - The minimum value of the range
+  * - max
+    - The maximum value of the range
+  * - n
+    - The number of people, households or other units in the range
+  * - moe
+    - The margin of error for the number of units in the range
+
+Also expects the number of simulations to run (default = 50) and an optional seed if you want your results to be replicable. 
+Note that due to the simulation, this function takes some time to run. The example takes a little over a minute. 
+Increasing the number of runs should increase your precision (with diminishing returns) at the cost of a slower run time.
+If you do not set the seed, another call of the function with the same inputs is not guaranteed to return the same answer.
```

Comment:
We should explicitly say this method was not developed using Census Bureau guidelines and give an outside reference.